core/src/main/kotlin/org/jetbrains/kotlinx/dataframe/documentation/AccessApi.kt

```diff
@@ -81,8 +72,9 @@ internal interface AccessApi {
      *
      * For more information: {@include [DocumentationUrls.AccessApis.ExtensionPropertiesApi]}
      *
-     * For example: {@comment This works if you include the test module when running KoDEx}
+     * For example, in notebooks extension properties are generated from runtime data after the cell is executed: {@comment This works if you include the test module when running KoDEx}
```

Comment:
You can put the `@comment` on the next line, if you write it without `{}`, there should be no newline in between `executed:` and `@sample`